### DISCUSSION TITLE
请打钱 - Xheldon Blog

### DISCUSSION THREAD

**Discussion by Xheldon** [Xheldon/x_blog#118] [Announcements]
https://www.xheldon.com/donate/ 

 时间就是金钱，所以，请打钱

**Comment by Xheldon** [comment]
给我打赏的同学目前不少了，有些同学是在电报群里问问题感谢得到了解答，有的同学是感谢我翻译的 ProseMirror 中文文档。

在这里感谢以上同学！

建议打赏完后最好在这个评论区里 `留下大名` ，不然我可能都不知道是谁和为什么要给我打赏，我会针对给我最多给我打赏的原因，持续耕种，谢谢~🤣

**Reply by pangfeng** [reply]
文档做的很好，已打上~

**Comment by JackyFreedom** [comment]
兄弟，slate 自定义元素换行样式出问题怎么处理，有什么办法吗

**Reply by Xheldon** [reply]
看下代码，和出问题的截图；你这么说「出问题」，我咋知道出啥问题🤣
